client/src/components/HomePage/modal/JoinModal/index.jsx

```diff
@@ -23,7 +24,11 @@ class Join extends Component {
     this.setState({ signingUp: true });
     setTimeout(() => {
```

Comment:
@HemaSAli is this setTimeout still needed? I wonder if it was originally added to simulate the loading delay of the signup endpoint, before the endpoint was built?